L1Trigger/L1THGCal/src/HGCalTriggerGeometryGenericMapping.cc

```diff
@@ -111,6 +112,7 @@ bool HGCalTriggerGeometryGenericMapping::disconnectedModule(const unsigned modul
 
 unsigned HGCalTriggerGeometryGenericMapping::lastTriggerLayer() const {
   return eeTopology().dddConstants().layers(true);
+  // HFNOSE to update
```

Comment:
I think there is no need to implement HFNose here. This is an old geometry class.

Reply:
ok changes to this file removed